File: 2026POTD/JANUARY/25-01-2026/1984.cs
public class Solution {
    public int MinimumDifference(int[] nums, int k) {
        if (k <= 1) return 0;
        Array.Sort(nums);
        int ans = int.MaxValue;
        for (int i = 0; i + k - 1 < nums.Length; i++) {
            ans = Math.Min(ans, nums[i + k - 1] - nums[i]);
        }
        return ans;
    }
}

File: 2026POTD/JANUARY/25-01-2026/1984.ts
function minimumDifference(nums: number[], k: number): number {
    if (k <= 1) return 0;
    nums.sort((a, b) => a - b);
    let ans = Infinity;
    for (let i = 0; i + k - 1 < nums.length; i++) {
        ans = Math.min(ans, nums[i + k - 1] - nums[i]);
    }
    return ans;
}